docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -120,8 +121,8 @@ In the event of a total active region loss, the following data will be lost:
 
 ## Limitations
 
-- Camunda 8 must be installed with the [Camunda Helm chart](/self-managed/setup/install.md).
-  - Alternative installation methods (with docker-compose installation, for example) are not supported.
+- We are strongly recommending using Kubernetes dual-region setup, with [Camunda Helm chart](/self-managed/setup/install.md) installed into two Kubernetes clusters
```

Comment:
What is the meaning of `strongly recommended` vs `recommended`?

Reply:
This is to highlight that complexity of other installation methods is much higher. But fine by me to only leave `recommended`.

I've also searched the docs and we're having multiple cases of the `strongly recommended`, so it's not something totally new to use in this place.